### PR TITLE
backport: Use jolokia version defined in camel-k-runtime-bom (1.10)

### PR DESCRIPTION
**Pull request by claudio4j**
**Release Note**
```release-note
NONE
```

Depends on https://github.com/apache/camel-k-runtime/pull/964

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: pkg/trait/jolokia.go [hunk 1]
```diff
@@ -63,7 +63,7 @@ func (t *jolokiaTrait) Apply(e *Environment) error {
 		}
 
 		// TODO: We may want to make the Jolokia version configurable
```
Comment (squakez) [inline pkg/trait/jolokia.go:65]: nit: remove this TODO, also from main if it's still there (a follow up PR would be fine)

**Review by squakez — APPROVED** [review]

**Comment by squakez** [comment]
Let's rerun the check in a couple of hours, once runtime snapshot is published.

**Comment by squakez** [comment]
I've just realized we don't use, neither publish snapshot for release branch in ck runtime. Hence we can merge this.

**Comment by squakez** [comment]
I've just realized we don't use, neither publish snapshot for release branch in ck runtime. Hence we can merge this.

**Pull request merged**